js/blocks.js

```diff
@@ -224,7 +224,7 @@ class Blocks {
             }
 
             /** Force a refresh. */
-            await delayExecution(500);
+            await delayExecution(100);
```

Comment:
Have you taken confirmation with maintainers for this? Because the time for 500ms and 100ms seems quite less and there might be a reason for adding 500ms earlier I believe.

Reply:
We made some changes in the rendering logic, so this speed up seems to be OK.